tests/CMakeLists.txt

```diff
@@ -24,11 +24,19 @@ endforeach()
 foreach(PROG ${PROGS})
     foreach(DI fm bifm)
         foreach(MODE ${MODES})
-            foreach(FF m0 m8 m9 sam bam m9.gz sam.bz2)
-                add_test (NAME test_search_${PROG}_${DI}_${MODE}_${FF}
-                        COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/maintests.sh
-                    "${CMAKE_SOURCE_DIR}" "${CMAKE_BINARY_DIR}" ${PROG} ${DI} ${MODE} "SEARCH" ${FF})
-            endforeach()
+            if(CMAKE_SYSTEM_NAME STREQUAL Linux)
```

Comment:
Can you replace this with "not macOS" for now since the tests do work on FreeBSD?

Reply:
Yes no problem